Library/Formula/libantlr3c.rb

```diff
@@ -1,9 +1,12 @@
-require 'formula'
-
 class Libantlr3c < Formula
-  homepage 'http://www.antlr3.org'
-  url 'http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz'
-  sha1 'faa9ab43ab4d3774f015471c3f011cc247df6a18'
+  homepage "http://www.antlr3.org"
+  url "http://www.antlr3.org/download/C/libantlr3c-3.4.tar.gz"
+  sha1 "faa9ab43ab4d3774f015471c3f011cc247df6a18"
+  revision 1
+
+  option "with-exceptions", "Compile with support for exception handling"
+  option "with-antlrdebug", "Compile with antlr debugger in runtime"
```

Comment:
The exceptions one is new, and the antlrdebug one is so that cvc4 can depend on a version without that flag. Would you recommend making the option `--without-antlrdebug` instead?